src/background/container.ts

```diff
@@ -101,7 +106,12 @@ export const configureContainer = () => {
       })),
     message: asClass(MessageManager<BackgroundToContentMessage>).singleton(),
     tabEvents: asClass(TabEvents).singleton(),
-    sendToPopup: asClass(SendToPopup).singleton(),
+    sendToPopup: asClass(SendToPort, {
+      injector: () => ({ connectionName: BACKGROUND_TO_POPUP_CONNECTION_NAME }),
+    }).singleton(),
+    sendToApp: asClass(SendToPort, {
+      injector: () => ({ connectionName: BACKGROUND_TO_APP_CONNECTION_NAME }),
+    }),
```

Comment:
Nit: Let's use `inject` instead of `injector` like we do on rest of registrations, for consistency.

```suggestion
    sendToPopup: asClass(SendToPort)
      .singleton()
      .inject(() => ({
        connectionName: BACKGROUND_TO_POPUP_CONNECTION_NAME,
      })),
    sendToApp: asClass(SendToPort)
      .singleton()
      .inject(() => ({
        connectionName: BACKGROUND_TO_APP_CONNECTION_NAME,
      })),
```